cmd/nfd-master/main.go

```diff
@@ -91,6 +91,8 @@ func main() {
 			klog.InfoS("-port is deprecated, will be removed in a future release along with the deprecated gRPC API")
 		case "verify-node-name":
 			klog.InfoS("-verify-node-name is deprecated, will be removed in a future release along with the deprecated gRPC API")
+		case "enable-spiffe":
```

Comment:
Probably not. I think this will never be enabled by default so feature gate feels superfluous (and unintuitive). It would serve as a unnecessary second-level gate (we'd want the enable/disable setting anyway), i.e. you'd need to specify `-feature-gates spiffe=true -enable-spiffe`